app/src/main/res/values/dimens.xml

```diff
@@ -64,4 +63,10 @@
   <dimen name="recently_played_tablet_land_outer_margin">96dp</dimen>
   <dimen name="recently_played_tablet_port_outer_margin">120dp</dimen>
   <dimen name="default_reading_text_size">18sp</dimen>
+
+  <dimen name="font_scale_content_text_size">20dp</dimen>
+  <dimen name="profile_already_added_margin">20dp</dimen>
+  <dimen name="profile_already_added_margin_land">20dp</dimen>
```

Comment:
Shouldn't we just always be using ``profile_already_added_margin``? One of the benefits of using named dimensions is we can keep the same name & overwrite the value in landscape (e.g. with a ``values-land/dimens.xml`` file).

Reply:
Yes you are right, I will make a dimension **profile_already_added_margin** in the landscape version of dimens.xml

Reply:
Resolving this on behalf of @BenHenning